packages/react-instantsearch-hooks-web/src/widgets/__tests__/SortBy.test.tsx

```diff
@@ -20,11 +20,6 @@ describe('SortBy', () => {
       </InstantSearchHooksTestWrapper>
     );
 
-    await wait(0);
-
-    expect(document.querySelector('.ais-SortBy-select')).toHaveValue(
-      'instant_search'
-    );
```

Comment:
This expectation was removed?

Reply:
Oops, that was a mistake. Thanks for catching it!